src/subsplit_dag.cpp

```diff
@@ -143,8 +147,8 @@ size_t SubsplitDAG::GetRootsplitIndex(const Bitset &rootsplit) const {
   return RootsplitIndexOfId(subsplit_to_id_.at(rootsplit));
 }
 
-size_t SubsplitDAG::RootsplitIndexOfId(size_t root_id) const {
-  return dag_edges_.at({root_id, root_id});  // #273
+size_t SubsplitDAG::RootsplitIndexOfId(size_t rootsplit_id) const {
```

Comment:
I'm going to need to be convinced that this method needs to exist at all given the excellent addition of the DAG root!